src/transaction/log_page_buffer.c

```diff
@@ -1944,7 +1944,7 @@ logpb_copy_page (THREAD_ENTRY * thread_p, LOG_PAGEID pageid, LOG_CS_ACCESS_MODE
   if (get_server_type () == SERVER_TYPE_TRANSACTION && ats_Gl.is_page_server_connected ())
     {
       // wait for answer.
-      auto log_page = ats_Gl.get_log_page_broker ().wait_for_page (pageid);
+      auto log_page = ats_Gl.get_page_broker<log_page_type> ().wait_for_page (pageid);
 
       // Sould be the same.
```

Comment:
Obvious comment is obvious. :)

Reply:
Comment was there before the code...so I know what code to write :)